### ISSUE TITLE
Field should return size

### ISSUE THREAD

**Issue by jon-chuang** [arkworks-rs/algebra#56]
Field should return extension degree + characteristic (already impl) so that one can easily determine its order.

**Comment by ValarDragon** [comment]
I don't think we need the field to return the size (due to the representation problem), I view providing the extension degree and characteristic as sufficient.

Maybe we could add a `floor_field_size_in_bits`, by calculating `floor(extension_degree * log_2(characteristic))` with a logarithm method that returns a decimal. (Don't want to return float, due to non-determinism across arch's)
